web calculator/web calculator/index.html

```diff
@@ -0,0 +1,20 @@
+<html>
```

Comment:
You've got multiple folders nested here when you don't need any. Also good convention to get into, devs don't use spaces in file and folder names. Spaces in file/folder names cause issues on the terminal as well as in some automated tools. For that reason you'll never see them in code bases.